Makefile

```diff
@@ -69,7 +69,7 @@ e2e: e2e-clean e2e-setup e2e-run
 
 $(PROJECT_PATH)/_output/unit.cov:
 	mkdir -p "$(PROJECT_PATH)/_output"
-	go test ./pkg/... -v -covermode=count -test.coverprofile="$(PROJECT_PATH)/_output/unit.cov"
+	go test ./pkg/... -v -tags=unit -covermode=count -coverprofile $(PROJECT_PATH)/_output/unit.cov -coverpkg ./...
```

Comment:
I understand -tags=unit has been added just in case in the future we want to explicitely disallow one of the test files to be run by writing //+build !unit. If it's not the case please tell me what's the motivation. Thank you.

Reply:
The idea is to separate Go tests with Build Tags. Then you can filter.